mibs/YADRO-MIB.txt

```diff
@@ -35,8 +38,11 @@ yadro MODULE-IDENTITY
 yadroNotifications      OBJECT IDENTIFIER ::= { yadro 0 }
 yadroSensors            OBJECT IDENTIFIER ::= { yadro 1 }
 yadroConformance        OBJECT IDENTIFIER ::= { yadro 2 }
+yadroSoftware           OBJECT IDENTIFIER ::= { yadro 3 }
 yadroCompliances        OBJECT IDENTIFIER ::= { yadroConformance 1 }
 yadroGroups             OBJECT IDENTIFIER ::= { yadroConformance 2 }
+yadroSoftwareBMC        OBJECT IDENTIFIER ::= { yadroSoftware 1 }
+yadroSoftwareBIOS       OBJECT IDENTIFIER ::= { yadroSoftware 2 }
```

Comment:
BIOS is a bad name here. It's strictly x86. In OpenPOWER they call it "OpenPOWER Firmware". I'd replace `BIOS` with `HostFW` from "Host Firmware" or `OPFW` or just `PNOR`. My personal favourite for SNMP is `HostFW` because it is platform-independent and can be later reused in non-POWER systems.

This applies to all occurrences of `BIOS` in this pull request.